board/safety/safety_toyota.h

```diff
@@ -98,15 +96,13 @@ static int toyota_rx_hook(CANPacket_t *to_push) {
       }
     }
 
-    // sample speed
     if (addr == 0xaa) {
-      int speed = 0;
-      // sum 4 wheel speeds
-      for (uint8_t i=0U; i<8U; i+=2U) {
-        int wheel_speed = (GET_BYTE(to_push, i) << 8U) + GET_BYTE(to_push, (i+1U));
-        speed += wheel_speed - 0x1a6f;
+      // check any wheel speed for non-zero value
+      vehicle_moving = false;
```

Comment:
make sure this is done atomically

Reply:
Tried to just use `GET_BYTES_04`, but it weirdly returned `0x6f1a6f00` for 0 speed, not `0x6f1a6f1a`.